### PR TITLE
Transpile from tachyons-custom

### LINKED ISSUE #26: We should transpile tachyons-custom to expose more variables

### PR DESCRIPTION
**Pull request by ptrikutam**
As outlined in https://github.com/tachyons-css/tachyons-sass/issues/26, this PR aims to transpile `tachyons-custom` to provide more variables / customization options.

Changes: 

- Update the package.json to pull from `tachyons-custom` 4.9.1
- Update the `build.js` script to:
    - Update `tachyons` to `tachyons-custom` wherever appropriate
    - remove the conditions on line 17 that skip over `_colors` and `_media-queries` since those are both contained within `_variables`
    - Update the regex to add a space before the `Modules` comment matcher since that's how it is in the `tachyons-custom` repo.
- Removed the `_colors.scss` and `_media-queries.scss` files since those variables are available in the `_variables.scss` files now.

closes #26

### CODE REVIEW

File: scss/_heights.scss [hunk 1]
```diff
@@ -91,16 +91,16 @@
...
+  .h2-m { height: $height-2; }
+  .h3-m { height: $height-3; }
+  .h4-m { height: $height-4; }
+  .h5-m { height: $height-5; }
   .h-25-m { height: 25%; }
   .h-50-m { height: 50%; }
   .h-75-m { height: 75%; }
   .h-100-m { height: 100%; }
-  .min-h-100-m { min-height: 100%; }
+  .min-h-100-ns { min-height: 100%; }
```
Comment (johnotander) [inline scss/_heights.scss]: 🤔

File: scss/_heights.scss [hunk 2]
```diff
@@ -125,7 +125,7 @@
   .vh-50-l {  height:  50vh; }
   .vh-75-l {  height:  75vh; }
   .vh-100-l { height: 100vh; }
-  .min-vh-100-l { min-height: 100vh; }
+  .min-vh-100-m { min-height: 100vh; }
```
Comment (johnotander) [inline scss/_heights.scss]: 🤔

**Comment by johnotander** [comment]
You're the best! Looks like there might be a couple more bugs in `tachyons-custom`, but this is looking v good. Not sure if you're around to make the updates or not, I won't be this evening, but I can get around to it tomorrow.

And thanks for putting in all the hard work to get this going and flush some of these bugs out 💟

**Comment by ptrikutam** [comment]
@johnotander Thanks! You've been excellent at communicating throughout the whole process so I appreciate that.

I'm happy to make some more updates. Should I manually edit those files in `tachyons-custom` or is there some kind of transpiling process there too needs to happen?

File: scss/_heights.scss (hunk 1 above)
Comment (ptrikutam) [reply scss/_heights.scss]: Good catch!

**Comment by johnotander** [comment]
💚  `tachyons-custom` isn't currently transpiled so if you wanted to make the updates manually that'd be great. Thanks for your contributions!

**Comment by ptrikutam** [comment]
Here ya go: https://github.com/tachyons-css/tachyons-custom/pull/11

Once that's updated I'll update this PR.

**Comment by ptrikutam** [comment]
@johnotander ready for re-review.

**Comment by johnotander** [comment]
Looks great, thanks @ptrikutam !

**Pull request merged**